extensions/libraries/redcore/api/hal/hal.php

```diff
@@ -633,15 +633,18 @@ public function apiRead()
 
 		if ($displayTarget == 'list')
 		{
-			$functionName = RApiHalHelper::attributeToString($currentConfiguration, 'functionName', 'getItems');
+			$functionName  = RApiHalHelper::attributeToString($currentConfiguration, 'functionName', 'getItems');
+			$pFunctionName = RApiHalHelper::attributeToString($currentConfiguration, 'paginationName', '');
```

Comment:
set default value to 'getPagination' and please name variable to $paginationFunction :)
same with below line